packages/graphql/test/fixtures/fragments-with-special-characters/query.graphql

```diff
@@ -0,0 +1,7 @@
+#import "./(parentheses)/fragment.graphql"
+#import "./[brackets]/fragment.graphql"
+
+query Query {
+  ...ParenthesesFragment
+  ...BracketsFragment
```

Comment:
Brackets in filenames weren't actually breaking fragment imports, but I added this for extra confidence. Let me know if you'd like this to be removed.